README.md

```diff
@@ -83,27 +99,30 @@ const client = new MainClient({
   api_secret: API_SECRET,
 });
 
-client.getAccountTradeList({ symbol: 'BTCUSDT' })
-  .then(result => {
-    console.log("getAccountTradeList result: ", result);
+client
+  .getAccountTradeList({ symbol: 'BTCUSDT' })
+  .then((result) => {
+    console.log('getAccountTradeList result: ', result);
   })
-  .catch(err => {
-    console.error("getAccountTradeList error: ", err);
+  .catch((err) => {
+    console.error('getAccountTradeList error: ', err);
   });
 
-client.getExchangeInfo()
-  .then(result => {
-    console.log("getExchangeInfo inverse result: ", result);
+client
+  .getExchangeInfo()
+  .then((result) => {
+    console.log('getExchangeInfo inverse result: ', result);
   })
-  .catch(err => {
-    console.error("getExchangeInfo inverse error: ", err);
+  .catch((err) => {
+    console.error('getExchangeInfo inverse error: ', err);
   });
 ```
 
 See [spot-client.ts](./src/main-client.ts) for further information.
 
 ### REST USD-M Futures
-Start by importing the spot client. API credentials are optiona, though an error is thrown when attempting any private API calls without credentials.
+
+Start by importing the usd-m client. API credentials are optional, though an error is thrown when attempting any private API calls without credentials.
```

Comment:
No matter how many times I review this, there's always a typo somewhere 😅 good catch